.github/ISSUE_TEMPLATE/feature_request.md

```diff
@@ -27,9 +27,9 @@ assignees: ''
      a new feature.
 
      Is your feature request related to a problem? Please give a clear and
-     concise description of what the problem is.
+     a concise description of what the problem is.
```

Comment:
Grammatically I think the original was correct, there is a single description, and it should be both "clear" and "concise".

```suggestion
     concise description of what the problem is.
```

Reply:
Yea You are so right 👍